mods/tuxemon/l18n/cs_CZ/LC_MESSAGES/base.po

```diff
@@ -113,9 +113,33 @@ msgstr "Běž {name}!"
 msgid "combat_opponent_call_tuxemon"
 msgstr "{user} posílá ven {name}!"
 
+msgid "combat_state_poison_get"
```

Comment:
I think this will confuse weblate. You only need to add new stuff  to en_US. Translaters will fill in the other languages via weblate.